app/views/course/statistics/aggregate/course_performance.json.jbuilder

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+has_personalized_timeline = current_course.show_personalized_timeline_features?
+is_course_gamified = current_course.gamified?
+course_videos = current_course.videos
+course_video_count = course_videos.exists? ? course_videos.count : 0
+show_video = course_videos.exists? && can?(:analyze_videos, current_course)
+no_group_managers = @service.no_group_managers?
+
+json.hasPersonalizedTimeline has_personalized_timeline
+json.isCourseGamified is_course_gamified
+json.showVideo show_video
+json.courseVideoCount course_video_count
+json.hasGroupManagers !no_group_managers
+
```

Comment:
You can add additional information such as total achievements, total submissions, max level too! Perhaps these can be grouped together and called it as metadata.